.github/pull_request_template.md

```diff
@@ -1,37 +1,17 @@
-<!---
-Pull Request Template
-- Please fill the sections below as described in the comments.
-- This information simplifies collaboration and helps future readers.
-- Reviewers should ask for this info to be filled when missing.
--->
-
-<!---
-PR Metadata
-- Title: Provide a short summary of the PR changes.
-- Assignees: Assign yourself and other direct collaborators.
-- Type: Create the PR as a draft. Only set it to non-draft when it is ready to be reviewed.
-- Reviewers: Only assign reviewers when the PR is ready to be reviewed.
-- Labels: Only assign labels with high priority and important context. Default is label-less.
+<!--
+- Please fill the sections below and the PR metadata.
+- Make it readable to reviewers and future visitors.
```

Comment:
```suggestion
- Make it readable for reviewers and future visitors.
```

Reply:
this one is honestly such a minor change that is even debatable, they are both grammatically correct but I think you may have meant to use this one